dev/Telemetry/RuntimeProfiler.h

```diff
@@ -51,4 +62,14 @@ namespace RuntimeProfiler
         } \
     }
     
+#define __RP_Marker_ClassMemberById(typeindex, memberindex) \
```

Comment:
>__RP_Marker_ClassMemberById [](start = 8, length = 27)

macros everywhere else are in all caps and do not start with ___. Not sure why this is an exception.

Reply:
We could change it, although then we should change all of these in RuntimeProfiler.h, and then it'd deviate from the similar library in WUX.

Reply:
We are not trying to be consistent with WUX here. So i would vote to make it consistent with the rest of MUX codebase. @jevansaks ?